contracts/CollectiveGovernance.sol

```diff
@@ -131,31 +183,27 @@ contract CollectiveGovernance is Governance, VoteStrategy, ERC165 {
 
     /// @notice configure an existing proposal by id
     /// @param _proposalId The numeric id of the proposed vote
-    /// @param _quorumThreshold The threshold of participation that is required for a successful conclusion of voting
+    /// @param _quorumRequired The threshold of participation that is required for a successful conclusion of voting
     /// @param _requiredDuration The minimum time for voting to proceed before ending the vote is allowed
     function configure(
         uint256 _proposalId,
-        uint256 _quorumThreshold,
+        uint256 _quorumRequired,
         uint256 _requiredDuration
-    ) external {
+    ) external requireSupervisor(_proposalId) {
         address _sender = msg.sender;
-        _storage.revertNotValid(_proposalId);
-        revertNotSupervisor(_proposalId);
-
-        _storage.setQuorumRequired(_proposalId, _quorumThreshold, _sender);
         _storage.setVoteDuration(_proposalId, _requiredDuration, _sender);
-        _storage.makeFinal(_proposalId, _sender);
-        emit ProposalOpen(_proposalId);
+        configure(_proposalId, _quorumRequired);
     }
 
-    /// @notice open an existing proposal by id
+    /// @notice start the voting process by proposal id
     /// @param _proposalId The numeric id of the proposed vote
-    function openVote(uint256 _proposalId) external requireVoteReady(_proposalId) {
-        _storage.revertNotValid(_proposalId);
-        revertNotSupervisor(_proposalId);
-        revertVoteNotAllowed(_proposalId);
-
-        require(_storage.quorumRequired(_proposalId) < _storage.maxPassThreshold(), "Quorum required");
+    function startVote(uint256 _proposalId)
```

Comment:
why require a function to be called to open the vote? Why not just set a delay before voting starts and have it automatically open?

Reply:
There are a number of reasons for this I think.  The steps ensure the configuration is completed.  There is also symmetry (easier to understand) between the start() and end() and the end is required.   I like the idea that the state machine for voting is clear and driven by real world events not somehow 'assumed' based on circumstances.